composeApp/src/commonMain/kotlin/org/ooni/probe/domain/GetResults.kt

```diff
@@ -12,13 +12,13 @@ import org.ooni.probe.data.models.TestKeysWithResultId
 class GetResults(
     private val getResults: (ResultFilter) -> Flow<List<ResultWithNetworkAndAggregates>>,
     private val getDescriptors: () -> Flow<List<Descriptor>>,
-    private val getTestKeys: (ResultFilter) -> Flow<List<TestKeysWithResultId>>,
+    private val getTestKeys: (String?) -> Flow<List<TestKeysWithResultId>>,
```

Comment:
Not a bit thing, but next time use the `ResultModel.Id` as the argument, cause String can be anything.

Reply:
I am going to merge as is. It felt a bit weired passing `ResultFilter` as an argument and also instantiating `ResultModel.Id`. But i agree. Better to instantiate `ResultModel.Id`